server.py

```diff
@@ -346,6 +400,22 @@ def do_get_response(tx_id):
         return jsonify({}), 404
 
 
+@app.route('/size/<tx_id>', methods=['GET'])
```

Comment:
Can you remove this?  We don't have use for an endpoint that does this.  In order to test this new functionality, POST a submission to the /responses endpoint and then test the response (that should have the hash + the size)